messages/rcs/send-suggested-action-open-url.js

```diff
@@ -0,0 +1,53 @@
+require('dotenv').config({ path: __dirname + '/../../.env' });
+
+const VONAGE_API_KEY = process.env.VONAGE_API_KEY;
+const VONAGE_API_SECRET = process.env.VONAGE_API_SECRET;
+const VONAGE_APPLICATION_ID = process.env.VONAGE_APPLICATION_ID;
+const VONAGE_PRIVATE_KEY = process.env.VONAGE_PRIVATE_KEY;
+
+const TO_NUMBER = process.env.TO_NUMBER;
+const RCS_SENDER_ID = process.env.RCS_SENDER_ID;
+
+const { Vonage } = require('@vonage/server-sdk');
+const { RCSCustom } = require('@vonage/messages');
+const vonage = new Vonage({
+  apiKey: VONAGE_API_KEY,
+  apiSecret: VONAGE_API_SECRET,
+  applicationId: VONAGE_APPLICATION_ID,
+  privateKey: VONAGE_PRIVATE_KEY,
+});
+
+vonage.messages.send(
+  new RCSCustom({
+    custom: {
+      contentMessage: {
+        text: "Need some help? Call us now or visit our website for more information.",
+        suggestions: [
+          {
+            action: {
+              text: "Call us",
+              postbackData: "postback_data_1234",
+              fallbackUrl: "https://www.example.com/contact/",
+              dialAction: { 
+                phoneNumber: "+447900000000", 
+              },
+            },
+          },
+          {
+            action: {
+              text: "Visit site",
+              postbackData: "postback_data_1234",
+              openUrlAction: {
+                url: "http://example.com/", 
+              },
+            },
+          },
+        ],
+      },
```

Comment:
I think this might be a copy/paste error. It should be something like:

```suggestion
      contentMessage: {
        text: "Check out our latest offers!",
        suggestions: [
          {
            action: {
              text: "Open product page",
              postbackData: "postback_data_1234",
              openUrlAction: { 
                url: "http://example.com/", 
              },
            },
          },
        ],
      },
```